includes/class-wc-payments-account.php

```diff
@@ -372,6 +372,9 @@ public function get_account_status_data(): array {
 				'declineOnAVSFailure' => $account['fraud_mitigation_settings']['avs_check_enabled'] ?? null,
 				'declineOnCVCFailure' => $account['fraud_mitigation_settings']['cvc_check_enabled'] ?? null,
 			],
+			'campaigns'             => [
+				'wporgReview2025' => $account['eligibility_wporg_review_campaign_2025'] ?? false,
```

Comment:
This value is set to false by default so if the flag doesn't exist (or is removed from the server) the campaign will not be active.

Reply:
This is worthy of a code comment → added in 26ab129